homeassistant/components/nextdns/system_health.py

```diff
@@ -0,0 +1,25 @@
+"""Provide info to system health."""
+from __future__ import annotations
+
+from typing import Any
+
+from nextdns.const import API_ENDPOINT
+
+from homeassistant.components import system_health
+from homeassistant.core import HomeAssistant, callback
+
+
+@callback
+def async_register(  # pylint:disable=unused-argument
```

Comment:
We can remove the pylint disable. It's already globally disabled.

https://github.com/home-assistant/core/blob/148035c8ca8c6d706ec2ddcc9a52475b086ccb0d/pyproject.toml#L182

Reply:
https://github.com/home-assistant/core/pull/74503